### ISSUE TITLE
APA referencing style

### ISSUE THREAD

**Issue by CasparJungbacker** [TeachBooks/TeachBooks#8]
`/teachbooks/plugins` contain some code to enable APA-like referencing style in Jupyter Book. This stems from the time where `teachbooks` was meant to be some sort of everything-containing package. Really, the APA stuff should be a stand-alone package.

**Comment by rlanzafame** [comment]
sounds like the ideal thing is a sphinx plugin. once the new tool is ready we can delete the stale code and close this issue.

**Comment by CasparJungbacker** [comment]
I remember that Sphinx itself uses another package for formatting references, I think it was called `pybtex` or something. That package also supports a plugin system, so that's probably your best bet.

**Comment by rlanzafame** [comment]
yeah, `pybtext` takes care of most of the work, and `pybtexapastyle` is the plugin that Thirza added to the hydrology book. The local extension was how she then got the `()` brackets to work. So maybe it is a Python package rather than a sphinx extension. I'll let someone with more expertise decide :)
I reproduced the implementation in the manual in this PR https://github.com/TeachBooks/manual/pull/38
